src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: LayoutNG
+subhead: |
+  A new layout engine for Chromium
+authors:
+  - eae
+date: 2019-05-29
+hero: hero.jpg
+alt: hand sketched diagram of Chrome's paint and render pipeline
+description: |
+  LayoutNG is a new layout engine for Chromium that has been designed for the needs of modern scalable web applications. It provides improved performance isolation, better support for scripts other than Latin, fixes many issues around floats/margins and a large number of web compatibility issues.
+tags:
+  - post
+  - layout
+---
+
+<style>
+  .w-figures-2-up {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+  }
+</style>
+
+Scheduled to be released in Chrome 76, LayoutNG is a new layout engine exiting a multi-year effort. While this article intends to cover the list of immediate effects, longer term goals exist which intend to bring additional performance gains and advanced layout features.
+
+<h2 class="w-headline--four">What's New?</h2>
+
+1. Provides **improved performance** isolation
+1. **Better support** for scripts other than Latin
+1. **Fixes** many issues around floats and margins
+1. **Fixes** a large number of web compatibility issues
+
+{% Aside %}
+  Please note that LayoutNG will be **launched in stages**. In Chrome 76, LayoutNG is used for inline and block layout. Other layout primitives (such as table, flexbox, grid, and block fragmentation) will be replaced in subsequent releases.
+{% endAside %}
+
+<br>
+
+## Developer visible changes
```

Comment:
My thinking was that since there's only one h2, it's not really helping structure the text--and it's causing a bunch of h4s, which contribute to the wall-of-text look since they have less whitespace around them. Removing it would allow everything to be bumped up a hierarchy level.